[doc] Troubleshooting circular import (#2077)
taichi-dev/taichi: docs/faq.rst

**A:** Yes, as long as that *package* provides an interface with ``numpy``, see :ref:`other_python_packages`.

**Q:** Can we dynamically create fields within kernel / after kernel invocation?

> **A:** Sorry, we don't support that yet. Consider allocate a large enough field
>        in advance to walkaround.

Reviewer: ```suggestion
**A:** Sorry, we don't support it yet. Try allocating a large enough field
       in advance to walkaround.
```

**A:** You may export it with :ref:`export_ply_files` so that you could view it in Houdini or Blender.

>        Or make use the extension library `Taichi THREE <https://github.com/taichi-dev/taichi_glsl>`_ to render images and update to GUI in real-time.
>        Or make use our Blender addon `Taichi Blend <https://github.com/taichi-dev/taichi_blend>`_ to update meshes to Blender and make physically-based animations.

Reviewer 2: ```suggestion
       Or make use of the extension library `Taichi THREE <https://github.com/taichi-dev/taichi_glsl>`_ to render images and update to GUI in real-time.
       Or make use of our Blender addon `Taichi Blend <https://github.com/taichi-dev/taichi_blend>`_ to update meshes to Blender and make physically-based animations.
```
